README.md

```diff
@@ -1,2 +1,108 @@
-# ch-kafka-email
-Library for building and sending emails to chs-email-sender
+# CH Kafka Email
+
+Library for handling the production and sending of emails via Apache Kafka and chs-email-sender.
+
+This extends the CH Kafka library to allow the consistent creation and sending of emails via Kafka.
+
+## Requirements
+
+In order to run the service locally you will need the following:
+- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+
+## Getting Started
+
+The library is built using maven:
+```
+mvn clean install
+```
+
+The library can be imported as a maven dependency:
+```
+<dependency>
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>ch-kafka-email</artifactId>
+    <version>1.0.0</version>
+</dependency>
+```
+
+## Environmental Variables
```

Comment:
This is misleading. Some might think these are runtime env variables while they are just used by pipeline at build time. I even wonder whether these should be in README do to law significance.

Reply:
Sure, I'll remove